src/content/docs/contributing/nf_core_basic_training.md

```diff
@@ -214,6 +220,83 @@ Open ./modules/local/demo/module.nf and start customising this to your needs whi
 
 ### Making a remote module for a custom script
 
+To generate a module for a custom script you need to follow the same steps when adding a remote module. Then, you can supply the command for your script in the `script` block but your script needs to be present and *executable* in the `bin` folder of the pipeline. In the nf-core pipelines, this folder is in the main directory and you can see in [`rnaseq`](https://github.com/nf-core/rnaseq). Let's look at a example in this pipeline, for instance [`tximport.r`](https://github.com/nf-core/rnaseq/blob/master/bin/tximport.r). This is an Rscript present in the [`bin`](https://github.com/nf-core/rnaseq/tree/master/bin) of the pipeline. We can find the module that runs this script in [`modules/local/tximport`](https://github.com/nf-core/rnaseq/blob/master/modules/local/tximport/main.nf). As we can see the script is being called in the `script` block, note that `tximport.r` is being executed as if it was called from the command line and therefore needs to be *executable*.
+
+Let's create a simple custom script that converts a MAF file to a BED file called `maf2bed.py`:
+
+```
+#!/usr/bin/env python
+"""
+Author: Raquel Manzano - @RaqManzano
+Script: Convert MAF to BED format keeping ref and alt info
+"""
+import argparse
+import pandas as pd
+
+
+def argparser():
+    parser = argparse.ArgumentParser(description="")
+    parser.add_argument("-maf", "--mafin", help="MAF input file", required=True)
+    parser.add_argument("-bed", "--bedout", help="BED input file", required=True)
+    parser.add_argument(
+        "--extra", help="Extra columns to keep (space separated list)", nargs="+", required=False, default=[]
+    )
+    return parser.parse_args()
+
+
+def maf2bed(maf_file, bed_file, extra):
+    maf = pd.read_csv(maf_file, sep="\t", comment="#")
+    bed = maf[["Chromosome", "Start_Position", "End_Position"] + extra]
+    bed.to_csv(bed_file, sep="\t", index=False, header=False)
+
+
+def main():
+    args = argparser()
+    maf2bed(maf_file=args.mafin, bed_file=args.bedout, extra=args.extra)
+
+
+if __name__ == "__main__":
+    main()
+
+```
+
+Now, let's go back to our module:
```

Comment:
```suggestion
Since `maf2bed.py` is in the `bin` directory we can directory call it in the script block of our new module [module name] . You only have to be careful with how you call variables (some explanations on when to use `${variable}` vs. `$variable`):
```